### DISCUSSION TITLE
a problem  about q-table sort

### DISCUSSION THREAD

**Discussion by glIrving** [quasarframework/quasar#13582] [General - Components / Directives / etc]
When using the q-table, how can I only sort when fixing the first column so that he does not participate in the sorting，（by the way，i use the Official example ）

**Comment by MilosPaunovic** [comment]
In column definition ([https://quasar.dev/vue-components/table#defining-the-columns](https://quasar.dev/vue-components/table#defining-the-columns)) you can set `sortable` property to `false` if you don't want it to be sortable, if that's what you're asking.

**Reply by glIrving** [reply]
I mean when you select a different column to sort, the first column (or other column) is fixed and doesn't sort. Set stable=false to control one column not to sort, when other columns are clicked, all columns will follow the sort

**Reply by MilosPaunovic** [reply]
If what you're asking if there is an option to sort by multiple columns simultaneously - no, that's currently not possible out of the box.

**Reply by glIrving** [reply]
nonono,now i have 5 columns(A-E), i want to click  column B  , and then B-E is sorted, column A is fixed

**Reply by MilosPaunovic** [reply]
We should wait for someone else to weigh in as I'm still having trouble understanding what is it that you need.

**Reply by glIrving** [reply]
In simple terms, I want to fix one column without participating in the sort while clicking on other columns to sort
